custom_components/unifiprotect/binary_sensor.py

```diff
@@ -59,6 +59,7 @@ class UnifiProtectBinaryEntityDescription(
         name="Door",
         device_class=DEVICE_CLASS_DOOR,
         ufp_device_type=DEVICE_TYPE_SENSOR,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
Generally we don't want the `category` if its the primary function of the device since it means these entities will be excluded by default from being shown.

Reply:
Makes sense. I will remove it again.